types/pino-http/index.d.ts

```diff
@@ -20,6 +20,7 @@ declare namespace PinoHttp {
         genReqId?: GenReqId;
         useLevel?: Level;
         stream?: DestinationStream;
+        customLogLevel?: (res: unknown, error: unknown) => Level
```

Comment:
Is there a better type to use for `res`? I wasn't sure. The Pino docs don't say anything. 🤷‍♂️ 

If the module is used via `express-pino-logger`, it would be an `express.Request`. But what about other non-Express types?

Reply:
It's an [ServerResponse](https://nodejs.org/api/http.html#http_class_http_serverresponse)